README.md

```diff
@@ -5,3 +5,17 @@
 ## Overview
 
 A VSCode Extension for Seneca College devs.
+
+## Installation
+
+1. Install the dependencies:
+
+```
+npm i
+```
+
+2. cd to project folder
+
+## Usage
+
+To see the application running type F5 with your keyboard in any file of the project. Just make sure your mouse is in any of files in the project. A new window will pop up with a hello message. You are now in debugging mode. You can debug the code to find out what it is doing with the step forward step inward step out of buttons. Or press the red square to exit debugging mode.
```

Comment:
This still doesn't make sense to me.

Imagine I do exactly what you just said:

1. I run `npm i` to install the dependencies (I'm in a terminal)
2. Now I `cd` into the project folder.  This seems odd, since the project folder should be where I do `npm i`
3. Now I press F5 in my terminal?  That doesn't do anything

Reply:
Checking out package.json scripts is probably a good place to start:  https://github.com/Seneca-CDOT/vscode-seneca-college/blob/57ce8f85935d30ad1bdfda96bc4bfc6a4d80e48c/package.json#L33-L43

Reply:
> 

I fixed it. Let me know if it is better.